packages/pwa-kit-react-sdk/src/ssr/server/react-rendering.js

```diff
@@ -256,6 +256,8 @@ const renderApp = (args) => {
 
     const helmet = Helmet.renderStatic()
 
+    const isProduction = process.env.DEPLOY_TARGET === 'production'
```

Comment:
Arguably you would not want stack traces for staging, or any other environment that might contain keys or other sensitive data in the stack trace, not just production....

Reply:
```
const isProduction = process.env.NODE_ENV === 'production'
```

Reply:
As @olibrook mentioned below, you could also use `isRemote` it might make mocking easier and makes it semantically more correct in the sense that we are going to want this behaviour for all deployed environments (remote might sound better than using production).

Reply:
@bendvc That is a much weaker protection – if you want a guarantee, you want to do `isRemote()`.

Personally, I'd avoid `NODE_ENV` because the team is already pretty confused on exactly what should be targeted here:

`NODE_ENV` means _"a developer built this bundle with `NODE_ENV` set to production on their machine"_. They can then choose to deploy that bundle to any target they want. And that'd would allow people to revealing those stack-traces on a legitimate production site.

Using `isRemote()` is correct, in all cases - you'd see stack-traces on a developer machine and never see them on any bundle deployed to the Runtime.

Reply:
@vcua-mobify I have updated the GUS ticket for this to better capture that stack trace data should not be shown when the application is running somewhere publicly accessible, as opposed to in managed runtime environments named "production".

Semantically, we probably want a flag labelled something like "includeStackTraceData" that defaults to false, and is set true when we know you are running in development.

Reply:
PR has been updated to use `isRemote()` and includes a test.